comfy_cli/test_cmdline.py

```diff
@@ -0,0 +1,26 @@
+from typer.testing import CliRunner
```

Comment:
I added test on `--workspace` as well on https://github.com/Comfy-Org/comfy-cli/pull/110.

Maybe merge my PR first and remove this test file? I also fixed the issue on mutually exclusive test between test runs.